src/main/resources/application.yml

```diff
@@ -22,3 +22,9 @@ security:
 
 axiell:
   url: ${COLLECTIONS_PATH_STAGE}
+  username: ""
+  password: ""
+
+kerberos:
+  realm: ${KERBEROS_REALM}
+  kdc: ${KERBEROS_KDC}
```

Comment:
Kanskje legge til i README at disse er nødvendig for at appen skal kunne brukes lokalt?